pygfx/geometries/_cylinder.py

```diff
@@ -143,7 +143,53 @@ def cylinder_geometry(
     theta_length=np.pi * 2,
     open_ended=False,
 ):
-    """Create geometry representing a cylinder."""
+    """Generate a cylinder or a cylinder segment.
+
+    .. @almarklein @korjin: isn't this inconsistend
+    .. with our convnetion that the z-axis is forward?
+
+    This function generates a cylinder or a cylinder segment. The cylinder's
+    axis runs along the local z-axis, and its midpoint is located at the local
+    origin. The cylinder's faces (top and bottom cap) are approximated by
+    regular N-sided polygons, with corners on a circle of the given radius.
```

Comment:
>  The cylinder's axis runs along the local z-axis

I don't think this is the case. I ran the `speres.py` example, and modify the geometry like this:
```py
geometry.positions.data[:,1] *= 0.5
```
Causing the spheres to be flat in the direction of their axis.
<img width="587" alt="image" src="https://user-images.githubusercontent.com/3015475/212287443-d6345118-6834-4e33-98b4-d8b618345551.png">

Reply:
I got this from the source of the cylinder, which constructs the mantle positions using the following segment:

https://github.com/pygfx/pygfx/blob/eb085bd5010445ea7ddc807da0d4e7a012a6798c/pygfx/geometries/_cylinder.py#L36-L39

I didn't do any testing on this though, so it could be that there is a rotation somewhere that matches it with our convention.

Reply:
Oh, but the cylinders are indeed not facing up:
<img width="481" alt="image" src="https://user-images.githubusercontent.com/3015475/212412973-3f76228c-fd9a-45b8-b804-2cce7996993a.png">

I think this is something that we should change.

Reply:
Let's discuss this in https://github.com/pygfx/pygfx/issues/436